src/data-generator/users.ts

```diff
@@ -289,7 +290,8 @@ export async function createUsers(
   for (let i = fieldAgents.length; i < config.fieldAgents; i++) {
     fieldAgents.push(
       await createUser(token, randomOffice.id, {
-        role: 'FIELD_AGENT'
+        role: 'FIELD_AGENT',
+        type: 'FIELD_AGENT'
```

Comment:
Had to define the `type` field because of the UI giving an error without it: https://github.com/opencrvs/opencrvs-core/issues/2639